packages/renderer/MdToHtml/rules/mermaid.ts

```diff
@@ -98,17 +98,6 @@ export default {
 
 const exportGraphButton = (ruleOptions: RuleOptions) => {
 	const theme = ruleOptions.theme;
-	// Clicking on export button manually triggers a right click context menu event
-	const onClickHandler = `
```

Comment:
This `onClick` handler was converted from an inline event handler to an event handler added by a script.